client/web/src/insights/components/form/repositories-field/components/flex-textarea/FlexTextarea.module.scss

```
@@ -0,0 +1,6 @@
+.textarea {
+    margin: 0;
+    overflow: auto; // for ie, https://css-tricks.com/textarea-tricks/#article-header-id-8
```

```suggestion
    overflow: auto; // for ie, https://css-tricks.com/textarea-tricks/#9-remove-default-scrollbars-in-internet-explorer
```

Nice! Didn't know we supported ie

---

This component came from my personal pet project where I'm trying to support IE. I'll remove this rule and comment, thanks